src/wallet/wallet2.cpp

```diff
@@ -6301,8 +6359,9 @@ void wallet2::store_to(const std::string &path, const epee::wipeable_string &pas
         LOG_ERROR("error removing file: " << old_mms_file);
       }
     }
-  } else {
-    // save to new file
+  } // end `if (!same_file)`
```

Comment:
This end comment is not necessary and breaks convention from the rest of the code. Personally I find it distracting.